shenyu-admin/src/main/java/org/apache/shenyu/admin/controller/NamespacePluginController.java

```diff
@@ -55,7 +55,7 @@
 /**
  * this is namespace plugin controller.
  */
-@RestApi("/namespacePlugin")
+@RestApi("/namespace_plugin")
```

Comment:
You should keep consistent with other places. e.g. `namespace-plugin` would be better.

Reply:
sounds good